bin/cron/Z_sheet.rb

```diff
@@ -0,0 +1,99 @@
+#1) API-key: 0b35d4f4184c255ce002643252f0671e
```

Comment:
Whoops!  It looks like some additional files got pushed to this PR.  Also, is this API key a secret?